Cargo.toml

```diff
@@ -95,12 +95,13 @@ vector-api-client = { path = "lib/vector-api-client", optional = true }
 vrl-cli = { path = "lib/vrl/cli", optional = true }
 
 # Tokio / Futures
-async-trait = "0.1"
+async-trait = "0.1.42"
 futures = { version = "0.3", default-features = false, features = ["compat", "io-compat"] }
 futures01 = { package = "futures", version = "0.1.25" }
-tokio = { version = "0.2.13", features = ["blocking", "fs", "io-std", "macros", "process", "rt-core", "rt-threaded", "signal", "stream", "sync", "time", "udp", "uds"] }
-tokio-openssl = "0.4.0"
-tokio-util = { version = "0.3.1", features = ["codec"] }
+tokio = { version = "1.3.0", features = ["full"] }
```

Comment:
I'll review if we need the `"full"` feature here.

Reply:
Talked to @lukesteensen: _if_ it's worth investigating, we'll move it to another issue.